metrics/social-currency-metric-system.md

```diff
@@ -1,20 +1,19 @@
-## Archive note 
-
-This metric was archived due to its impracticality in a business environment. As noted by a maintainer: business investment is made from a relative value. While there is a lot of great information in this metric, there is no reason for our target audience to go to this level of trouble to show reputation.
-
-Note: if you find some portion fo this infromation useful, please add it to an existing metric or offer a new one that is more aligned to our mission. 
-
 # Social Currency Metric System (SCMS)
 
 Question: How does one measure the value of community interactions and accurately gauge “reputation” of a community as evident from qualitative sentiment?
 
-
 ## Description
 
+SCMS is a combination of [social listening](https://blog.hubspot.com/service/social-listening) practices across multiple channels along with a meaningful set of categorizations. The combination of these these tactics can lead to systematic community analysis and can inform a community strategy that leads to direct business value.
+
+### Theory and origin
```

Comment:
We want to limit the template headings to the ones we predetermined.
For metrics like SCMS that would benefit from additional sub-headings, we use bolded word, rather than actual subheadings. I know it's quirky but it is the compromise we currently have to keep our metrics simple.

```suggestion
**Theory and Origin**
```